### ISSUE TITLE
Module not found error

### ISSUE THREAD

**Issue by che-122** [opendevtools/supreme-ts#24]
`supreme` commands do not work. Getting an error related to a path that is misconfigured or missing:

`Error: Cannot find module '../dist/'`

```bash
➜ node -v
v12.9.1
➜ npm -v
6.10.2
```

**Comment by che-122** [comment]
Tested in Node LTS as well (10.16.3). Getting the same error.

**Comment by lessp** [comment]
Can reproduce:

```sh
❯ npx @iteam/supreme reason supreme-test-reason
npx: installed 128 in 6.312s
Cannot find module '../dist/'
Require stack:
- /Users/ter/.npm/_npx/7879/lib/node_modules/@iteam/supreme/bin/supreme
```

**Comment by believer** [comment]
Should be fixed by 8d8067e55db70c84f9dd40ad5fa3e1d0dfb285b6. Can you verify @faroutchris @lessp?

**Comment by believer** [comment]
Verified working by @Jimjardland
